src/main/java/com/ucm/ms/branches/entities/Appointment.java

```diff
@@ -0,0 +1,103 @@
+package com.ucm.ms.branches.entities;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Objects;
+
+@Entity
+@Table(name = "appointment")
+public class Appointment implements Serializable {
+	private static final long serialVersionUID = -2548083122650649843L;
+
+	// Data
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Integer id;
+
+	@Column(name = "scheduled_time")
+	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss[.SSS][.SS][.S]")
+	private LocalDateTime scheduledTime;
+
+	@Column(name = "service")
+	private String service;
+
+	// Relationships
+	@ManyToOne
+	@JoinColumn(name = "branch_location_number", referencedColumnName = "location_number")
+	private Branch branch;
+
+	@ManyToOne
+	@JoinColumn(name = "banker_id", referencedColumnName = "id")
+	private Banker banker;
+
+	// Methods
+	public Appointment(LocalDateTime scheduledTime, String service, Branch branch, Banker banker) {
+		super();
+		this.scheduledTime = scheduledTime;
+		this.service = service;
+		this.branch = branch;
+		this.banker = banker;
+	}
```

Comment:
When defining a constructor for an entity class, it should have a no-arg public constructor overload as per JPA specifications.